flo-runner/src/test/java/com/spotify/flo/context/Mocks.java

```diff
@@ -0,0 +1,209 @@
+/*-
+ * -\-\-
+ * Flo Runner
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.flo.context;
+
+import com.google.common.collect.ImmutableList;
+import com.spotify.flo.EvalContext;
+import com.spotify.flo.FloTesting;
+import com.spotify.flo.Task;
+import com.spotify.flo.TaskContextStrict;
+import com.spotify.flo.TestContext;
+import com.spotify.flo.status.NotReady;
+import java.net.URI;
+import java.util.ArrayDeque;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.atomic.AtomicInteger;
+
+@SuppressWarnings("WeakerAccess")
+public class Mocks {
```

Comment:
It would be nice to expose these mocks to used by the end users as well, aside from the internal tests? Like in a `flo-test` module or as part of the `flo-runner`

Reply:
I do not imagine that users will use these mocks of imaginary storage systems. Instead, users will want to mock inputs and outputs for actual production systems. E.g. in our case we want mocks for hades, bigquery, gcs, etc.

Reply:
Thanks for clarifying, I had misunderstood the purpose of these mocks.